packages/expo-apple-authentication/src/AppleAuthentication.ts

```diff
@@ -83,7 +92,20 @@ export async function refreshAsync(
   return credential;
 }
 
-// @docsMissing
+// @needsAudit
+/**
+ * An operation that ends the authenticated session.
+ * Calling this method will show the sign in modal before actually signing the user out.
+ *
+ * It is not recommended to use this method to sign out the user as it works counterintuitively.
+ * Instead of using this method it is recommended to simply clear all the user's data collected
+ * while using [`signInAsync`](./#signinasync) or [`refreshAsync`](./#refreshasync) methods.
```

Comment:
```suggestion
 * from using [`signInAsync`](./#signinasync) or [`refreshAsync`](./#refreshasync) methods.
```

this reads slightly more clearly, because "while" in this case can be misinterpreted to mean in parallel with